rbc/stdlib/array_methods.py

```diff
@@ -0,0 +1,7 @@
+"""
+Array API specification for array methods.
+
+https://data-apis.org/array-api/latest/API_specification/data_type_functions.html
+"""
+
+__all__ = []
```

Comment:
Why is this file needed? Isn't it already `rbc/stdlib/data_type_functions.py`?

Reply:
Sort of. I still need to include the code for unimplemented methods and attributes.

Reply:
ok thanks for confirming, I was not sure as the PR was not in draft.

Reply:
Sorry about that. In the end I remove this file and moved the attributes to `rbc/omnisci_backend/omnisci_array.py`